src/workers/postAnalytics/postAnalyticsUpdate.ts

```diff
@@ -0,0 +1,27 @@
+import { PostAnalytics } from '../../entity/posts/PostAnalytics';
+import type { TypedWorker } from '../worker';
+
+export const postAnalyticsUpdate: TypedWorker<'api.v1.post-metrics-updated'> = {
+  subscription: 'api.post-analytics-update',
+  handler: async ({ data }, con): Promise<void> => {
+    const { postId, payload } = data;
```

Comment:
should we use zod to validate the payload?

Reply:
yeah we can, actually `parseMessage` argument on worker object makes it easy 👌